modules/op-chains-generating-cosign-secret.adoc

```diff
@@ -0,0 +1,70 @@
+// This module is included in the following assemblies:
+// * secure/using-tekton-chains-for-openshift-pipelines-supply-chain-security.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="chains-generating-cosign-secret_{context}"]
+= Generating the cosign key pair by using the TektonConfig CR
+
+To use the `cosign` signing scheme for {tekton-chains} secrets, you can generate a `cosign` key pair that uses ECDSA encryption by setting the `generateSigningSecret` field in the `TektonConfig` custom resource (CR) to `true`.
+
+.Prerequisites
+
+* You installed the {oc-first} utility.
+* You are logged in to your {OCP} cluster with administrative rights for the `openshift-pipelines` namespace.
+
+.Procedure
+
+. Edit the `TektonConfig` CR by running the following command:
++
+[source,terminal]
+----
+$ oc edit TektonConfig config
+----
+
+. In the `TektonConfig` CR, set the `generateSigningSecret` value to `true`:
++
+.Example of creating an ECDSA cosign key pair by using the TektonConfig CR
```

Comment:
Since I've been tagged here, I tested the file to ensure it does convert to DITA without any issues and it does. The only thing that gets flagged during conversion is this block title which is assigned to a code block, something that DITA has no equivalent markup for. DITA does allow titles to be assigned to example blocks (would need to be marked up as such in AsciiDoc with the `[Example]` attribute headr or the `====` delimiter), but only allows one example in the main body of the task, not nested within a step.

Reply:
Thanks! This is a good question but it's commonly used in docs, including RNs, so we'll have to come up with *some* way of converting them - or drop the titles on a case-by-case basis. This one can probably be dropped when moving to DITA. While we're on Asciidoc I want this one to be consistent with the rest of the docs.

Reply:
I absolutely agree, and this is not a blocker and we have plenty of time to figure out what to do with titles like these, I just highlighted this for awareness. The module builds and the conversion tool falls back to using a paragraph with bold text and assigned output class to clearly identify where that text came from.